src/feedback/commands/submitFeedback.ts

```diff
@@ -68,15 +73,15 @@ export async function submitFeedbackMessage(
         })
     } catch (err) {
         const errorMessage = (err as Error).message || 'Failed to submit feedback'
-        logger.error(`Failed to submit ${message.sentiment} feedback: ${errorMessage}`)
+        logger.error(`feedback failed: "${message.sentiment}": ${errorMessage}`)
```

Comment:
Our webview code already logs communication (but at debug level) so this and line 64 can be dropped.